avocado_virt/defaults.py

```diff
@@ -30,6 +30,11 @@
         qemu_bin = 'qemu'
 
 try:
+    no_kvm = settings.get_value('virt.qemu', 'no_kvm')
+except SettingsError:
+    no_kvm = False
```

Comment:
I'm not sure whether this option is so important to deserve a setting and argument. Do you think people would need to change it frequently? I consider it as a special case so I'd avoid the whole defaults and cmdline arguments and only respect the value if present.

@apahim, @clebergnu what are your opinions?